ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -189,3 +189,33 @@ def test_numpy_rfftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.rfft2",
+    dtype_input_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
+        shape=(2,2),
+        min_num_dims=0,
+        max_num_dims=2,
```

Comment:
Since the array has to be 2D you will need to set `min_num_dims=2` as well. And no need to hardcode the `shape`. The `shape` could be of any 2D size, not just `(2,2)`.